src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:33691/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:33691/project/issues?id=sample&pullRequest=5454&issues=AWcSOdLH4-O_2Fwwu23z&open=AWcSOdLH4-O_2Fwwu23z)

[ISSUE_KEY]: AWcSOdLH4-O_2Fwwu23z